cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) Compliance in CockroachDB Cloud Dedicated
+summary: Learn about compliance with the Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Cloud Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a series of requirements for the safe handling of sensitive data associated with credit and debit cards. In the PCI DSS standard, and in the rest of this page, this data is referred to as "cardholder data". When implemented correctly, PCI DSS helps to protect your organization and your customers from fraud. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
+
+{% include v22.1/dedicated-pci-compliance.md %}
+
+Responsibility for compliance with PCI DSS is shared among multiple parties, including card issuers, banks, software-as-a-service (SaaS) providers, and retail merchants. Compliance measures are implemented as a series of business practices, security controls, and technological solutions. An organization's compliance with PCI DSS is certified by a PCI Qualified Security Assessor (QSA).
```

Comment:
Perhaps at some point we'll want to link from here to a doc about the process for requesting NDA for access to our audit reports and compliance resources? Or even just say ask your account team?